zktest/test/java/org/zkoss/zktest/zats/test2/F85_ZK_1148Test.java

```diff
@@ -45,11 +45,22 @@ public void testOutWindowDetach() {
 
 	@Test
 	public void testDesktopDestroy() {
+		F85_ZK_1148FileDealer.writeMsg("", false);
 		DesktopAgent da = connect();
 		da.destroy();
-		Assert.assertEquals("DestroyD is called!DestroyC is called!", F85_ZK_1148FileDealer.readMsg());
+		Assert.assertEquals("DestroyD is called!DestroyC is called!IncludeVM is called!", F85_ZK_1148FileDealer.readMsg());
```

Comment:
https://github.com/zkoss/zk/pull/1185/files#diff-c52ff1da920c31e56cabddf44f726869R34
You forgot the potential issue

Reply:
ok thanks